### PR TITLE
Update blogs.yml

### PR DESCRIPTION
**Pull request by lavavi**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

- Replace this with your change description

### CODE REVIEW

**Comment by lavavi** [comment]
blog

**Comment by nzec** [comment]
@lavavi Please tick all the boxes that you have completed. Also please provide a live link using Github Pages.
(If you don't know how to do that just give a screenshot).

**Comment by lavavi** [comment]
Done
screenshot_20171210-125552

**Review by akshatnitd — CHANGES_REQUESTED** [review]
@lavavi Please add the blog image in the PR :)

File: _data/blogs.yml [hunk 1]
```diff
@@ -47,3 +47,8 @@
   title: Interview of a Student participating in the Google Code In
   img: 6.png
   author_name: Hasalu Botheju
+
+- url: https://bloglavavi.wordpress.com/
+  title: My First Experience of Google Code - In
+  img: BLOG.jpg
+  author_name: Name of the author of the blog (most likely you)
```
Comment (akshatnitd) [inline _data/blogs.yml]: Add you name here

**Comment by lavavi** [comment]
I have given it via chat....

File: _data/blogs.yml [hunk 2]
```diff
@@ -47,3 +47,8 @@
   title: Interview of a Student participating in the Google Code In
   img: 6.png
   author_name: Hasalu Botheju
+
+- url: https://bloglavavi.wordpress.com/
+  title: My First Experience of Google Code - In
+  img: BLOG.jpg
```
Comment (akshatnitd) [inline _data/blogs.yml:53]: Add this image in ```/img/blog/``` folder and add it in the commit

**Review by akshatnitd — CHANGES_REQUESTED** [review]
Please do the requested changes :)

**Comment by lavavi** [comment]
when i am trying to upload the file its showing 
The Uploads are disabled.
File uploads require push access to this repository.

**Comment by vaibhavsingh97** [comment]
@lavavi You don't need to upload your image directly to this repository. Add the image in your local repo and then push the changes to your `origin` and changes will be reflected here.

**Pull request closed without merging**